doc/man3/BIO_f_ssl.pod

```diff
@@ -240,6 +240,13 @@ a client and also echoes the request to standard output.
      ERR_print_errors_fp(stderr);
      exit(1);
  }
+ /*Added BIO_do_accpet cal after first BIO has been setup*/
```

Comment:
the formatting should habe a space between /* and text, as well before */
furthermore typo at BIO_do_acc**pe**t and **cal**.

I think the comment could point out that the first BIO_do_accept is actually a bind and/or listen
while the second BIO_do_accept is really an accept (if I understood the code at all)

Reply:
Thanks for comment and now updated the comments properly and waiting for my CLA approval

Reply:
No problem, good to hear you are willing to send a ICLA.
BTW: cal is wrong english, it need to be spelled ca**ll**

Reply:
> Thanks for comment and now updated the comments properly and waiting for my CLA approval

I assume you mean approval internal to Intel - I've not seen your CLA come through to us yet.

Reply:
Yes waiting for approval from Internal to Intel.